lib/src/cmaf/utils/hls/HLSToHam.ts

```diff
@@ -0,0 +1,179 @@
+import { parseM3u8 } from '../hls/m3u8.js';
+import { uuid } from '../../../utils.js';
+import {
+	AudioTrack,
+	TextTrack,
+	Segment,
+	SelectionSet,
+	SwitchingSet,
+	Track,
+	VideoTrack,
+} from '../../ham/types/model';
+import { addMetadataToHLS } from '../manifestUtils.js';
+import { PlayList } from '../../ham/types/mappers/HlsManifest.js';
+import { Manifest } from '../../ham/types/';
+
+function m3u8ToHam(manifest: Manifest) {
```

Comment:
I think we deleted this file because is not used. Can you confirm that?

Reply:
That´s right. Thank you for noticing it!  I had it in my local branch and didn´t update. Just deleted.